src/createServer.js

```diff
@@ -1,3 +1,33 @@
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+
+const http = require('http');
+const { getParams } = require('./ServerFunctions/getParams');
+const { getErrors } = require('./ServerFunctions/getErrors');
+const {
+  getSuccessfulResponse,
+} = require('./ServerFunctions/getSuccessfulResponse');
```

Comment:
create an index.js file in `./ServerFunctions` where you will import all of your functions and export them
then you could import them here via a single import